shapely/ctypes_declarations.py

```diff
@@ -545,3 +545,8 @@ def prototype(lgeos, geos_version):
 
         lgeos.GEOSMinimumClearance.argtypes = [c_void_p]
         lgeos.GEOSMinimumClearance.restype = c_double
+
+    if geos_version > (3, 8, 0):
```

Comment:
should this have been '>='?

Reply:
Yep, sure should have been. See #930.

Reply:
The MR was just merged, so you should hopefully be good to go. Thanks again for reporting this.

Reply:
Thanks for making this PR!